src/Bookings.js

```diff
@@ -12,10 +12,10 @@ const Bookings = () => {
     <div className="App-content">
       <div className="container">
         <Search search={search} />
-        {/* <SearchResults results={FakeBookings} /> */}
+        <SearchResults results={FakeBookings} />
```

Comment:
Typically we want to keep non-component imports pascal case 
```suggestion
        <SearchResults results={fakeBookings} />
```